engine-docker/pom.xml

```diff
@@ -35,27 +35,38 @@
     </properties>
 
     <dependencies>
-
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-handler</artifactId>
         </dependency>
-
         <dependency>
             <groupId>com.github.docker-java</groupId>
             <artifactId>docker-java-api</artifactId>
         </dependency>
         <dependency>
             <groupId>com.github.docker-java</groupId>
             <artifactId>docker-java-core</artifactId>
-            <version>3.3.1</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
Same as above. We've worked to avoid guava dependencies due to painful integration problems in the past. If it's back in for some reason, let's see what we can do to limit the impact/exposure.

Reply:
@jshook looking at the dependencies, it looks like `
com.github.docker-java:docker-java-core@3.3.1` brings in this vulnerability for `guava` in nb.  

It's considered a low `cve`, so if we want to remove the direct dependency upgrade for `guava` we can keep the `low` marker on the security dashboard.  

Otherwise, the changes for the referenced `guava` version in the PR will remove that low vulnerability identified as: CWE-379: Creation of Temporary File in Directory with Insecure Permissions.

Reply:
😄 @jshook Looks like another bump of that `docker-java-core` just came in today [here](https://github.com/nosqlbench/nosqlbench/pull/1391). 

Let me refactor those dependencies once again...

Reply:
Updated latest with the recent upgrade for `docker-java-core` which has reported to handle much of these specific versions in the original push.